readme.md

```diff
@@ -0,0 +1,69 @@
+# Hackathon Project: SenaiLabIntegration
+
+## Overview
+
+Welcome to the GitHub repository for our project developed during the 20-hour InovaTI HACKATHON. This project aims to integrate a laboratory testing machine into the lab's digital system.
+
+## The Challenge
+
+During the Hackathon, participants were tasked with creating innovative solutions for real problems local companies have. Our team focused on solving the data integrity problem of a lab test result. Senai, a local lab, tests construction materials such as concrete to verify their quality. After testing the material, a lab technician has to read the result value on the machine's display and record the result in their system, which is error-prone.
+
+## Our Solution
+
+The **lab-integration** tracks a material testing machine's test result and automates a form submission for a new result on the lab's website. We wanted to write directly on the database, but the lab's IT department refused access. As we didn't know the database schema, we had to build a macro to fill the form on the lab's website, which the lab technician let us skim through.
+
+The **mock-testing-environment** mocks the lab's environment: running a test and recording the test result.
```

Comment:
Do you think that is necessary mention the lab-integration branch here too?

Reply:
It's already there.